client/landing/site.landing/coffee/team/forms/teamdomaintabform.coffee

```diff
@@ -19,42 +19,34 @@ module.exports = class TeamDomainTab extends kd.FormView
       click        : => @input.setFocus()
 
     @inputView.addSubView @input = new kd.InputView
-      placeholder  : 'your-team'
-      attributes   : 10
+      placeholder  : 'Your team name...'
```

Comment:
yeah, the team name's input size was dynamically calculated in the past to center the whole field. That `10` was `size: 10` if it is not the case anymore removing them is just fine.